schema/askbot.csv

```diff
@@ -1,4 +1,5 @@
-name,type
+name,type
+accepted_answer_id,long

```

Comment:
i would rename this to `question_accepted_answer_id`. Same reason as below, to keep the same convention.